Datadog/Datadog.xcodeproj/project.pbxproj

```diff
@@ -3591,9 +3591,9 @@
 /* End PBXHeadersBuildPhase section */
 
 /* Begin PBXNativeTarget section */
-		61133B81242393DE00786299 /* Datadog */ = {
+		61133B81242393DE00786299 /* Datadog iOS */ = {
```

Comment:
I'm wondering if we should use whitespaces in target names. As long as we properly `""` handle it in our `xcconfigs`, scripts and tools, there should be no problem. Problems may arise if we miss it somewhere or one of our 3rd-party integration tools does (Cocoapods / Carthage). Calling it `Datadog-iOS` could be the same good, but save us some trouble (in theory). WDYT @maxep ?

Reply:
🤔 It's pretty common to use spaces in targets (see PLCR), I find it more human readable in Xcode. I guess it will be easy to catch if we miss `""` somewhere in our automation.
CP will create its own targets based on the podspec, and Carthage is based on [shared schemes](https://github.com/Carthage/Carthage#share-your-xcode-schemes), so I think we are good. Tho I'm open to reconsidering if we have some friction in maintenance, now that the product name does not depend on the target, it will be trivial to change. How does it sounds?

Reply:
Totally fine 👌. I was wondering if you see this dangerous as well or I'm just over cautious. Good to use spaces and see how it goes 👌.